Vagrantfile

```diff
@@ -16,13 +16,28 @@ Vagrant.configure(2) do |config|
   config.vm.provider "virtualbox" do |vb|
     # Customize the amount of memory on the VM:
     vb.memory = "2048"
+    vb.cpus = 4
   end
 
   config.vm.provision "shell", inline: <<-SHELL
      sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
```

Comment:
Maybe we could strip down provisioning to `curl https://get.docker.com | sh` + opening port 2375.